src/adaptors/in-memory-articles.ts

```diff
@@ -1,48 +1,44 @@
-import { Iri, JsonLdObj } from 'jsonld/jsonld-spec';
-import { schema } from 'rdf-namespaces';
+import {
+  BlankNode, DatasetCore, Quad, Quad_Object as QuadObject,
```

Comment:
Rename is a little annoying, eslint complains even though it's defined outside...

Reply:
Tried setting `'@typescript-eslint/camelcase': ['error', { ignoreImports: true }]`, stops the import being flagged as an error but any further usage still is...

Reply:
Wonder why just this name follows a different style (than `BlankNode` for example)?

Reply:
https://gitter.im/rdfjs/public?at=5e0e07da04a1900497f07c9e (not sure why)